src/Features/Core/Portable/QuickInfo/Presentation/TaggedTextExtensions.cs

```diff
@@ -0,0 +1,280 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Runtime.CompilerServices;
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Shared.Collections;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.QuickInfo.Presentation;
+
+internal static class TaggedTextExtensions
```

Comment:
📝 This was moved and refactoted from https://github.com/dotnet/roslyn/blob/main/src/EditorFeatures/Core/IntelliSense/Helpers.cs.